### PR TITLE
KEYCLOAK-13712 Keycloak Operator Quickstart Guide

### PR DESCRIPTION
**Pull request by slaskawi**
https://issues.redhat.com/browse/KEYCLOAK-13712

Getting Started Guide might be found here: https://github.com/keycloak/keycloak-web/pull/124

### CODE REVIEW

File: operator-examples/myrealm.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+apiVersion: keycloak.org/v1alpha1
+kind: KeycloakRealm
+metadata:
+  name: myrealm
+  labels:
+    realm: myrealm
+spec:
+  realm:
+    id: "myrealm"
```
Comment (stianst) [inline operator-examples/myrealm.yaml:9]: Why are you setting "id" here? This is really an internal ID and shouldn't be set here.

File: operator-examples/myrealm.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    realm: "myrealm"
+    enabled: True
+    displayName: "myrealm"
```
Comment (stianst) [inline operator-examples/myrealm.yaml:12]:
```suggestion
    displayName: "My Realm"
```

File: operator-examples/myrealm.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+kind: KeycloakRealm
+metadata:
+  name: myrealm
+  labels:
+    realm: myrealm
+spec:
+  realm:
+    id: "myrealm"
+    realm: "myrealm"
+    enabled: True
```
Comment (stianst) [inline operator-examples/myrealm.yaml:11]: A shame this is needed, as it really should be enabled by default ;)

File: operator-examples/keycloak.yaml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+apiVersion: keycloak.org/v1alpha1
+kind: Keycloak
+metadata:
+  name: mykeycloak
+  labels:
+    app: sso
```
Comment (stianst) [inline operator-examples/keycloak.yaml:6]: Would it be a bit more clear if the label was something more like "mykeycloak"?

File: operator-examples/myuser.yaml [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+  labels:
+    app: sso
+spec:
+  user:
+    username: "myuser"
+    firstName: "John"
+    lastName: "Doe"
+    email: "myuser@keycloak.org"
+    enabled: True
+    emailVerified: False
```
Comment (stianst) [inline operator-examples/myuser.yaml:14]: I'd remove this

File: operator-examples/myuser.yaml [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+    credentials:
+      - type: "password"
+        value: "12345"
```
Comment (stianst) [inline operator-examples/myuser.yaml:17]: There's something a bit odd about having credentials in clear-text here. It's fine for now, but I think this means we need to consider how/if users are provisioned as CRs.

File: operator-examples/myrealm.yaml (hunk 1 above)
Comment (slaskawi) [reply operator-examples/myrealm.yaml:9]: Removed.

File: operator-examples/myrealm.yaml (hunk 2 above)
Comment (slaskawi) [reply operator-examples/myrealm.yaml:12]: Fixed.

File: operator-examples/keycloak.yaml (hunk 4 above)
Comment (slaskawi) [reply operator-examples/keycloak.yaml:6]: Updated.

File: operator-examples/myuser.yaml (hunk 5 above)
Comment (slaskawi) [reply operator-examples/myuser.yaml:14]: Done

File: operator-examples/myuser.yaml (hunk 6 above)
Comment (slaskawi) [reply operator-examples/myuser.yaml:17]: +1 for the future consideration.

File: operator-examples/myrealm.yaml (hunk 3 above)
Comment (slaskawi) [reply operator-examples/myrealm.yaml:10]: +1

**Pull request merged**